google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerExceptionFactory.java

```diff
@@ -48,6 +50,11 @@ public static SpannerException propagateInterrupt(InterruptedException e) {
     return SpannerExceptionFactory.newSpannerException(ErrorCode.CANCELLED, "Interrupted", e);
   }
 
+  public static SpannerException propagateTimeout(TimeoutException e) {
```

Comment:
Where is this supposed to be used?

Reply:
This is not used in the client library but by people who [depend on spanner inside google3](https://cs.corp.google.com/piper///depot/google3/java/com/google/corp/services/facilities/cloudpoc/dao/spanner/utils/DatabaseSchemaUpdater.java?sq=package:piper+file://depot/google3+-file:google3/experimental&g=0&l=71).

Reply:
got it.  Please add a comment to this method, and it might be good to give an example use.